reinforcement_learning/examples/rl_sim_cpp/rl_sim_cpp.vcxproj

```diff
@@ -180,6 +180,7 @@
     <Import Project="..\..\..\vowpalwabbit\packages\boost.1.63.0.0\build\native\boost.targets" Condition="Exists('..\..\..\vowpalwabbit\packages\boost.1.63.0.0\build\native\boost.targets')" />
     <Import Project="..\..\..\vowpalwabbit\packages\boost_program_options-vc140.1.63.0.0\build\native\boost_program_options-vc140.targets" Condition="Exists('..\..\..\vowpalwabbit\packages\boost_program_options-vc140.1.63.0.0\build\native\boost_program_options-vc140.targets')" />
   </ImportGroup>
+  <Import Project="$(VcpkgIntegration)" Condition="Exists('$(VcpkgIntegration)')" />
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
Can we add an error if VcpkgIntegration doesnt exist?

Reply:
I would rather not, because that would force everyone to use it, rather than enabling them to use the more common way of using vcpkg, via the "vcpkg integrate" automagic.

Reply:
I see, not a problem. I agree.

---
In reply to: [217426031](https://github.com/JohnLangford/vowpal_wabbit/pull/1599#discussion_r217426031) [](ancestors = 217426031)